subsystem/cluster_resource_rosa_test.go

```diff
@@ -2026,7 +2035,7 @@ var _ = Describe("rhcs_cluster_rosa_classic - create", func() {
 		`)
 		Expect(terraform.Apply()).To(BeZero())
 	})
-	It("Creates cluster with aws subnet ids & private & private link", func() {
+	It("Creates private cluster with aws subnet ids & private link", func() {
```

Comment:
I think we missed here the test (that should be failed) not Private with private link